paper/paper.md

```diff
@@ -1,351 +1,436 @@
+```python
+from PySDM import Formulae
+from PySDM.physics import si
+from PySDM_examples.Singer_Ward.aerosol import AerosolBetaCaryophylleneDark
+aerosol = AerosolBetaCaryophylleneDark()
+formulae_bulk = Formulae(surface_tension='Constant')
+formulae_ovad = Formulae(
+    surface_tension='CompressedFilmOvadnevaite',
+    constants={
+        'sgm_org': 35 * si.mN / si.m,
+        'delta_min': 1.75 * si.nm
+    }
+)
+formulae_ruehl = Formulae(
+    surface_tension='CompressedFilmRuehl',
+    constants={
+        'RUEHL_nu_org': aerosol.aerosol_modes[0]['nu_org'],
+        'RUEHL_A0': 115e-20 * si.m * si.m,
+        'RUEHL_C0': 6e-7,
+        'RUEHL_m_sigma': 0.3e17 * si.J / si.m**2,
+        'RUEHL_sgm_min': 35 * si.mN / si.m
+    }
+)
+formulae_sl = Formulae(
+    surface_tension='SzyszkowskiLangmuir',
+    constants={
+        'RUEHL_nu_org': aerosol.aerosol_modes[0]['nu_org'],
+        'RUEHL_A0': 115e-20 * si.m * si.m,
+        'RUEHL_C0': 6e-7,
+        'RUEHL_sgm_min': 35 * si.mN / si.m
+    }
+)
+```
 
+Using these different models for the surface-partitioning, we can demonstrate the effect variable surface tension has on the activation of aerosol with some organic fraction. The presence of the orgnaics both modifies the surface tension and the hygroscopicity, resulting sometimes in a Köhler curve with local minima features. Below is (psuedo-)code used to generate four Köhler curves for the same partially organic aerosol particle, just under different assumptions of surface-partitioning by the insoluble organic species. 
 ```python
-from PySDM.physics.constants_defaults import rho_w
-from matplotlib import pyplot
-
-for step in [0, 1200, 2400, 3600]:
-    particulator.run(step - particulator.n_steps)
-    pyplot.step(
-        x=radius_bins_edges[:-1] / si.um,
-        y=particulator.products['dv/dlnr'].get().squeeze() * rho_w/si.g,
-        where='post', label=f"t = {step}s")
-
-pyplot.xscale('log')
-pyplot.xlabel(r'particle radius [$\mu$ m]')
-pyplot.ylabel("dm/dlnr [g/m$^3$/(unit dr/r)]")
-pyplot.legend()
-pyplot.show()
+import numpy as np
+for formulae in (formulae_bulk, formulae_ovad, formulae_ruehl, formulae_sl):
+    r_wet = np.logspace(np.log(50 * si.nm), np.log(2000 * si.nm), base=np.e, num=100)
+    sigma = np.ones(len(r_wet))
+    for j,vw in enumerate(formulae_ovad.trivia.volume(r_wet)):
+        sigma[j] = formulae.surface_tension.sigma(
```

Comment:
It's this `sigma` evaluation with the `CompressedFilmRuehl` that causes a Numba warning.

Reply:
For now, I updated `-We` to `-Wd` to allow warnings in the `joss_paper.yml` execution.